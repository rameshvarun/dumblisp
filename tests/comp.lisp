(print (= 3 0) (= 0 0) (= 1 1) (= 100 1) (= 50 50))
(print (> 3 0) (> 0 0) (> 1 1) (> 100 1) (> 50 50))
(print (< 3 0) (< 0 0) (< 1 1) (< 100 1) (< 50 50))
