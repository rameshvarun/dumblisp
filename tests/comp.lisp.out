FALSE TRUE TRUE FALSE TRUE
TRUE FALSE FALSE TRUE FALSE
FALSE FALSE FALSE FALSE FALSE
